app/models/configuration_profile.rb

```diff
@@ -69,7 +69,7 @@ def tag_hash
     @tag_hash ||= configuration_tags.index_by(&:class)
   end
 
-  alias_method :configuration_manager, :manager
+  alias configuration_manager manager
```

Comment:
think we want `alias_attribute` instead since `manager` is a `belongs_to`.

(again, up to you how to proceed with this)

Reply:
I'll back this one out too - we can deal in separate PRs.